CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.3.1 (2016-06-21)
```

Comment:
I was thinking this was a minor version bump, i.e. 1.4.0. I don't see this as a patch.